src/services/users/signin-user-by-email.test.js

```diff
@@ -0,0 +1,52 @@
+const { expect } = require('chai')
+const db = require('../../db')
+const {fixtures} = require('../../../test/utils')
+const signInUserByEmail = require('./signin-user-by-email')
+
+describe('User signin', () => {
+	let user
+	let validPassword
+	beforeEach(async () => {
+		user = await fixtures.insertUser({
+			password : 'validPaswword123$'
+		})
+		validPassword = 'validPaswword123$'
+	})
+
+	it('should throw an error if user does not exist', async () => {
+		const { id,email } = user
+		await db.query(
+			`DELETE FROM users
+             WHERE id = $1`, [id])
+
+		return expect(signInUserByEmail(email , validPassword))
+			.to.be.rejectedWith(
+				'Invalid email or password.')
+	})
+
+	it('should signin user', async () => {
+		const { id, email} = user
+		await signInUserByEmail(
+			email, validPassword)
+
+		const { rows } = await db.query(
+			`SELECT * FROM users
+             WHERE id = $1`, [id])
+
+		const  loggedInUser = rows[0]
+
+		return expect(loggedInUser).to.include({
```

Comment:
Are we trying to know if the signingUserEmail returns the right response? 

because this expectation is comparing the `loggedInUser` object with itself. Will always pass no matter what and we can't know if the function breaks for some reason

Reply:
i think you might be using an older commit oo, because this is the second thing that isn't in my code locally

Reply:
so what is in your `signin-user-by-email.test.js`